regolith/helpers/makeappointmentshelper.py

```diff
@@ -33,7 +33,10 @@
 
 
 def subparser(subpi):
-
+    # Do not delete --database arg
+    subpi.add_argument("--database",
```

Comment:
please put this at the bottom of the list so it appears last in the help.  It is rarely used (by design)

Reply:
done